### PR TITLE
fix: e2e

### PR DESCRIPTION
**Pull request by eugbyte**
attempt to fix the failing circle ci;. Failure seems to occur at `npm run build:umd step`, so i upgraded browserlist

```
> @govtechsg/decentralized-renderer-react-components@0.0.0-development build:umd
> webpack "--mode" "production"

Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
node:internal/crypto/hash:67
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
```

### CODE REVIEW

**Comment by ryanseee** [comment]
hmm seems like its still failing. I checked online, people mentioned might be something to do with the node version?

https://stackoverflow.com/questions/69692842/error0308010cdigital-envelope-routinesunsupported

**Comment by eugbyte** [comment]
https://github.com/webpack/webpack/issues/14532

Seems like when node version is upgrade to 17, `digital envelope routines::unsupported` error occurs with webpack

**Review by ryanseee — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ workflows:
 jobs:
   build:
     docker:
-      - image: 'circleci/node:latest'
+      - image: 'cimg/node:16.12.0'
```
Comment (Nebulis) [inline .circleci/config.yml:12]: use circleci/node:lts

**Review by Nebulis — CHANGES_REQUESTED** [review]
see comment

**Review by Nebulis — APPROVED** [review]

**Pull request merged**